src/index.html

```diff
@@ -4,9 +4,383 @@
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a href="#" class="header__logo-link">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link nav__link--active is-active">apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item" data-qa="nav-hover">
+            <a
+              href="#computers"
+              class="nav__link"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tabs" class="nav__link">планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="main-container">
+      <main class="main">
+        <section class="prd-card main__prd-card-container" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="apple a1419 imac 27 retina 5K"
+            class="prd-card__image"
+          >
+          <h1 class="prd-card__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h1>
+          <p class="prd-card__code">
+            Код товара: 195434
+          </p>
+          <div class="prd-card__rating">
+            <div class="stars">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+            <p class="prd-card__reviews">
+              Отзывов: 5
+            </p>
+          </div>
+          <div class="prd-card__price">
+            <p class="prd-card__price-label">
+              Цена
+            </p>
+            <p class="prd-card__price-number">
+              69 999 грн
+            </p>
+          </div>
+          <a
+            href="#"
+            data-qa="hover"
+            class="prd-card__buy"
+          >
+            купить
+          </a>
+        </section>
+        <section class="prd-card main__prd-card-container" data-qa="card">
```

Comment:
Don't use shortened words in class names.

```suggestion
        <section class="product-card main__product-card-container" data-qa="card">
```

Reply:
I've just corrected it, thank you :)